protos/telemetry/telemetry.proto

```diff
@@ -577,7 +577,11 @@ enum FixType {
 message Battery {
     uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
     float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
-    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0.0 to 1.0)
+    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0 to 100)
+    float current_battery_a = 4 [(mavsdk.options.default_value)="NaN"]; // Battery current in Amps, NAN if autopilot does not measure the current
```

Comment:
@julianoes The battery message itself will be a value in mA. If the conversion doesn't matter I'm fine with this being in Amps.

Reply:
You're referring to the mAh capacity of a battery. But here we are talking about Amps drawn from it, right?